### PR TITLE
Move sponsors of decommissioned machines to the 'emeritus' list

### PR DESCRIPTION
**Pull request by jimwins**

### CODE REVIEW

**Review by sy-records — APPROVED** [review]
Where did you find the list of decommissioned machine sponsors?

**Comment by jimwins** [comment]
> Where did you find the list of decommissioned machine sponsors?

https://wiki.php.net/systems (which I've also been updating based on removal of DNS records from the php/systems repo).

**Review by saundefined — APPROVED** [review]
I'm not sure about the relevance of sponsors on the wiki, so let's give some time for objections too, if there are any.

If no one objects, let's merge this, thanks!

**Review by derickr — COMMENTED** [review]
Left a comment about server central. I also agree with Sergey that we don't need to keep outdated information on the wiki. Of course, the most up to date one is *currently* at https://docs.google.com/document/d/19iRVGtr_g2kawQIZaPSeaEKfYjsirQ0MVB-NiZ9lWo0/edit

File: thanks.php [hunk 1]
```diff
@@ -64,78 +64,6 @@
...
-        <section>
-            <strong class="thanks__heading"><a href="https://rackspace.com">Rackspace</a></strong>
-            <p class="thanks__description">
-                Provides the server and bandwidth for various php.net services.
-            </p>
-        </section>
-    </li>
-
-    <li class="thanks">
-        <a href="https://www.servercentral.net" class="thanks__logo">
```
[the remark this answers is not in the document]
Comment (jimwins) [reply thanks.php:92]: Re-added with current name and logo

**Review by derickr — APPROVED** [review]
LGTM

**Pull request merged**